### PR TITLE
[#143584125] Add link to self-service password reset

### PR DESCRIPTION
**Pull request by chrisfarms**
# What

We have enabled the self service password reset feature and want to
bring it to the attention of tenants who may be looking for what to do if they have fotgotten their password.

# How to review

* Check it makes sense
* Preview it `bundle exec middleman server` 

## Who can review

not @chrisfarms or @henrytk

### CODE REVIEW

File: source/documentation/troubleshooting/password.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Forgetten passwords
```
Comment (keymon) [inline source/documentation/troubleshooting/password.md]: `s/Forgetten/Forgotten/`

Comment (henrytk) [reply source/documentation/troubleshooting/password.md]: Fixed. The link was also wrong. It now points to `/forgot_password`, not `/password_reset`.

**Comment by keymon** [comment]
:+1: LGTM 

Shall we wait for the feature to be in live before merging the docs?

**Comment by henrytk** [comment]
> Shall we wait for the feature to be in live before merging the docs?

Yes

**Pull request merged**